src/lib/SCEditor.js

```diff
@@ -1414,6 +1430,200 @@ define(function (require) {
 			});
 		};
 
+
+		/**
+		 * Adds a handle used to remap keys to make them more useful for code
+		 * input
+		 *
+		 * @private
+		 */
+		var addBodyCodeInputHandle = function (searchSelectors) {
+			$wysiwygBody.on('keydown', null,
+			{
+				'searchSelector': searchSelectors.join(',')
+			}, codeInputHandle);
+
+		};
+
+		/**
+		 * Adds a handle used to remap keys to make them more useful for
+		 * code input
+		 *
+		 * @private
+		 */
+		var codeInputHandle = function (e) {
+
+			if (e.ctrlKey || e.altKey || e.metaKey) {
+				return;
+			}
+
+			// For now, only Enter and Tab have defined executions
+			if (e.which !== 13 &&	// Enter
+				e.which !== 9		// tab
+			) {
+				return;
+			}
+
+			if (e.shiftKey && e.which !== 9) {
+				return;
+			}
+
+			rangeHelper.mergeTextNodesAtCaret();
```

Comment:
I don't think so. That code is only executed if all the above is false. These events are associated with certain HTML tags. In this case, this method must not be called as the selector (2nd parameter in the jQuery's .on()) is used.